### PR TITLE
Toggle trace/field visibility

### LINKED ISSUE #380: Feature: enable/disable graphs on click on "Legend graph description"
It would be very helpful for analysis if it is possible to activate or deactivate a special graph if user click onto the graph name in the legen without to change every time the Graph-Setup

BBE_Feature_GrapheActivate

### PR DESCRIPTION
**Pull request by atomgomba**
Fixes #380. I removed the equalizer icon from the legend because it had no real function and clicking on the field name had the same functionality (opening the analyzer display). Instead I replaced that icon with a toggle button to show/hide individual traces within a graph. The field names are now underlined on hover to indicate that they are clickable and open the analyser display just like before.

Screenshot from 2023-10-19 14-44-46

### CODE REVIEW

**Comment by nerdCopter** [comment]
i like this; however, would like color bar and possibly the number still be clickable for showing/updating the spectral graph

**Comment by nerdCopter** [comment]
unsure if this is possible to fix, but i also notice that occasionally, but not 100% of the time, the legend bars disappear during screen-update upon clicking an eye.
here i screen-recorded, then panned to a frame to show the disappearance which is only for milliseconds.
image

https://github.com/betaflight/blackbox-log-viewer/assets/56646290/6bf9155e-2677-41fd-8710-2ffa3063ed21

**Comment by atomgomba** [comment]
> unsure if this is possible to fix, but i also notice that occasionally, but not 100% of the time, the legend bars disappear during screen-update upon clicking an eye. here i screen-recorded, then panned to a frame to show the disappearance which is only for milliseconds. image
> 
> https://youtu.be/nU5Xhy7arOU

Thanks for noticing this! Yea, it's fixable; oh the joys of CSS!

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by atomgomba** [comment]
> unsure if this is possible to fix, but i also notice that occasionally, but not 100% of the time, the legend bars disappear during screen-update upon clicking an eye. here i screen-recorded, then panned to a frame to show the disappearance which is only for milliseconds. image
> 
> https://youtu.be/nU5Xhy7arOU

Turns out it was the unnecessary re-rendering of some elements. I also made the other elements clickable to open the analyser.

File: js/graph_legend.js
Comment (haslinghuis) [inline js/graph_legend.js:159]: Maybe use toggleClass (with state param)

**Review by haslinghuis — APPROVED** [review]

**Pull request merged**
